src/components/layout.tsx

```diff
@@ -74,6 +75,8 @@ const LayoutScaffolding = ({
 
   const locale = useCurrentLocale() || localeConfig.DEFAULT_LOCALE;
 
+  logAmplitudePageView();
```

Comment:
just to double-check cause i'm not super familiar with the orgsite page structure, this only gets called once when the page loads and not on every render?